adal/src/main/java/com/microsoft/aad/adal/IJWSBuilder.java

```diff
@@ -46,6 +46,6 @@
      * @param x509Certificate X509 certificate
      * @return Signed JWT
      */
-    public String generateSignedJWT(String nonce, String submitUrl, RSAPrivateKey privateKey,
+    String generateSignedJWT(String nonce, String submitUrl, RSAPrivateKey privateKey,
```

Comment:
Ah, right! This is an interface, sorry! 
Do you need to make it protected? All abstract methods in a public interface are implicitly public, no?

---
In reply to: [65662506](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/652#discussion_r65662506) [](ancestors = 65662506,65642749)